### PR TITLE
fix the bug when the filePath contains backslash

### PR DESCRIPTION
**Pull request by 928PJY**
fix the bug when the filePath contains backslash
@hellosnow @Huachao @qinezh @vwxyzh

### CODE REVIEW

File: src/VscPreviewExtension/media/js/previewMatch.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ var lastLocation = 0;
 
 var fileName = document.body.childNodes[1].data;
 
+var fileNameEscape = replaceAllInString(fileName, "\\", "\\\\");
```
Comment (928PJY) [inline src/VscPreviewExtension/media/js/previewMatch.js]: yes, that would be better!

**Comment by hellosnow** [comment]
:shipit:

File: src/VscPreviewExtension/media/js/previewMatch.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ var lastLocation = 0;
 
 var fileName = document.body.childNodes[1].data;
```
Comment (hellosnow) [inline src/VscPreviewExtension/media/js/previewMatch.js]: >fileName [](start = 4, length = 8)

should it be filePath? filename shouldn't contain backslash?

**Comment by Huachao** [comment]
:shipit:

**Pull request merged**